controllers/postgres_controller.go

```diff
@@ -696,18 +698,23 @@ func (r *PostgresReconciler) updatePatroniConfig(ctx context.Context, instance *
 
 	r.Log.Info("Sending REST call to Patroni API")
 	pods := &corev1.PodList{}
+
+	roleReq, _ := labels.NewRequirement("spilo-role", selection.In, []string{"master", "standby_leader"})
```

Comment:
Why ignoring the error?

Reply:
In worst case this can cause a panic in line 704. :sweat_smile:

Reply:
Fixed